contrib/externs/maps/google_maps_api_v3.js

```diff
@@ -19962,7 +19962,7 @@ google.maps.places.Place.prototype.servesBeer;
 google.maps.places.Place.prototype.servesBreakfast;
 
 /**
- * Whether a place serves brunch. Returns &#39;true&#39; or &#39;false&#39; if
+ * Whether a place serves branch. Returns &#39;true&#39; or &#39;false&#39; if
```

Comment:
This was not a typo, the property is `google.maps.places.Place.prototype.servesBrunch;`